_inc/lib/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This is the base class for every Core API endpoint Jetpack uses.
+ *
+ */
+abstract class Jetpack_Core_API_XMLRPC_Consumer_Endpoint {
+
+	/**
+	 * An instance of the Jetpack XMLRPC client to make WordPress.com requests
+	 *
+	 * @private
+	 * @var Jetpack_IXR_Client
+	 */
+	private $xmlrpc;
```

Comment:
well, since I still haven't figured out what IXR means or why it's in the name, I think that gives a reasonable nudge that updating the name could help. I'm not much for naming after historical things that no longer exist. on the other hand, it's just confusion and doesn't need to block anything: I'm not sure how far-reaching a name-change would be.